Sources/Locker/Internal/LockerHelpers.swift

```diff
@@ -0,0 +1,184 @@
+        String(format: "%@_KeychainService", LockerHelpers.kBundleIdentifier)
+    }
+
+    // MARK: - User defaults keys help methods
+
+    static func keyKeychainAccountName(for uniqueIdentifier: String) -> String {
+        let kKeychainAccountName = String(format: "%@_KeychainAccount", locale: nil, kBundleIdentifier)
+        return String(format: "%@_%@", kKeychainAccountName, uniqueIdentifier)
+    }
+
+    static func keyDidAskToUserBiometricsID(for uniqueIdentifier: String) -> String {
+        let kUserDefaultsDidAskToUseBiometricsID = String(format: "%@_UserDefaultsDidAskToUseTouchID", locale: nil, kBundleIdentifier)
+        return String(format: "%@_%@", kUserDefaultsDidAskToUseBiometricsID, uniqueIdentifier)
+    }
+
+    static func keyBiometricsIDActivated(for uniqueIdentifier: String) -> String {
+        let kUserDefaultsKeyBiometricsIDActivated = String(format: "%@_UserDefaultsKeyTouchIDActivated", kBundleIdentifier)
+        return String(format: "%@_%@", kUserDefaultsKeyBiometricsIDActivated, uniqueIdentifier)
+    }
+
+    static func keyShouldAddSecretToKeychainOnNextLogin(for uniqueIdentifier: String) -> String {
+        let kUserDefaultsShouldAddSecretToKeychainOnNextLogin = String(format: "%@_UserDefaultsShouldAddPasscodeToKeychainOnNextLogin", kBundleIdentifier)
+        return String(format: "%@_%@", kUserDefaultsShouldAddSecretToKeychainOnNextLogin, uniqueIdentifier)
+    }
+
+
+    // MARK: - Private methods
+
+    private static func setLAPolicyDomainState(with domainState: Data?) {
+        UserDefaults.standard.set(domainState, forKey: LockerHelpers.keyLAPolicyDomainState)
+    }
+
+    // MARK: - Apiary
+
+    static func deviceSupportsAuthenticationWithBiometrics(_ completion: @escaping ((BiometricsType) -> Void)) {
+        LockerHelpers.deviceSupportsAuthenticationWithFaceID({ isSupported in
+            if isSupported {
+                completion(.faceID)
+            } else {
+                let context = LAContext()
+                var error: NSError? = nil
+
+                if !context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: &error) {
+                    // When user removes all fingers for TouchID, error code will be `notEnrolled`.
+                    // In that case, we want to return that device supports TouchID.
+                    // In case lib is used on simulator, error code will always be `notEnrolled` and only then
+                    // we want to return that biometrics is not supported as we don't know what simulator is used.
+                    if let error = error, error.code == kLAErrorBiometryNotAvailable || error.code == kLAErrorBiometryNotEnrolled && isSimulator {
+                        completion(.none)
+                    }
+                } else {
+                    completion(.touchID)
+                }
+            }
+        })
+    }
+
+    internal static var deviceList: DeviceList = .shared
+}
```

Comment:
In general, this file and the main Locker file should be better organized. Separate private or internal/public properties/methods into the logical `extension`/`private extensnion` groups, and keep (computed) properties before methods. Main logic should be first, then private methods and at the end, we should have helpers.